options/posix/generic/pwd-stubs.cpp

```diff
@@ -273,9 +273,18 @@ void endpwent(void) {
 	close_global_file();
 }
 
-int putpwent(const struct passwd *, FILE *) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+static bool invalid(const char *s) {
```

Comment:
Could you make this a local lambda in the function please? Avoids littering the global namespace with an unspecific name like ``invalid``

Reply:
done